{{cookiecutter.profile_name}}/slurm-submit.py

```diff
@@ -109,12 +110,22 @@
     if "{{cookiecutter.account}}" != "":
         arg_dict["account"] = "{{cookiecutter.account}}"
 
+# Ensure output folder for Slurm log files exist
+# This is a bit hacky; it will try to create the folder
+# for every Slurm submission...
+if "output" in arg_dict:
+    stdout_folder = Path(arg_dict["output"]).parent
+    stdout_folder.mkdir(exist_ok=True, parents=True)
+if "error" in arg_dict:
+    stdout_folder = Path(arg_dict["error"]).parent
+    stdout_folder.mkdir(exist_ok=True, parents=True)
```

Comment:
Please change these to use `os.makedirs`. By that, they become one-liners.

Reply:
Sorry. I just noticed I was looking at the wrong Python docs yesterday; `exist_ok` is indeed implemented for `os.makedirs` in Python 3.5+. I'll make the change!